torchvision/csrc/io/decoder/subtitle_stream.cpp

```diff
@@ -43,21 +43,34 @@ int SubtitleStream::initFormat() {
 int SubtitleStream::analyzePacket(const AVPacket* packet, bool* gotFrame) {
   // clean-up
   releaseSubtitle();
+
+  // FIXME: should this even be created?
```

Comment:
agree - I am also not sure if this should be created. Is the plan to follow up on this?

Reply:
Yes, but I think this would require a bit of coordination on Meta's side, as this is not used nor tested in TV.